library/src/main/java/com/northconcepts/templatemaster/form/CrudResource.java

```diff
@@ -51,6 +51,8 @@ public CrudResource(String singularTitle, String pluralTitle, String subUrl, For
 
     protected abstract Page<ENTITY> getPage(String keyword, String sortField, int pageNumber, Integer pageSize);
     
+    protected abstract Integer getCurrentUserPageSize();
```

Comment:
@gvjoshi25 we should rename this since -- from TemplateMaster's point of view -- we don't know if the page size is based on a user preference, system preference, or something else. 

```suggestion
    protected abstract Integer getCurrentPageSize();
```

Reply:
Made this change.